plugins/server/io.ktor/shutdown-url/2.0/manifest.ktor.yaml

```diff
@@ -1,5 +1,5 @@
 name: Shutdown URL
-description: This plugin enables a URL that when accessed, shutdowns the server.
+description: Enables a URL that when accessed, shuts down the server
```

Comment:
This would read better:
"Enables a URL that shuts down the server when accessed"